Tests/MeiliSearchUnitTests/UpdatesTests.swift

```diff
@@ -46,10 +70,51 @@ class UpdatesTests: XCTestCase {
             switch result {
             case .success(let result):
                 XCTAssertEqual(stubResult, result)
-                expectation.fulfill()
             case .failure:
                 XCTFail("Failed to get settings")
             }
+            expectation.fulfill()
+        }
+
+        self.wait(for: [expectation], timeout: 1.0)
+
+    }
+
+    func testGetUpdateInvalidStatus() {
+
+        //Prepare the mock server
+
+        let json = """
```

Comment:
```suggestion
        let badStatusUpdate = """
```
Should be changed in the example above aswell but I do not have access with github interface.